sdk/program/src/message/versions/mod.rs

```diff
@@ -28,7 +28,7 @@ pub const MESSAGE_VERSION_PREFIX: u8 = 0x80;
 /// which message version is serialized starting from version `0`. If the first
 /// is bit is not set, all bytes are used to encode the legacy `Message`
 /// format.
-#[frozen_abi(digest = "x2F3RG2RhJQWN6L2N3jebvcAvNYFrhE3sKTPJ4sENvL")]
+#[frozen_abi(digest = "G4EAiqmGgBprgf5ePYemLJcoFfx4R7rhC1Weo2FVJ7fn")]
```

Comment:
Just out of curiosity, how is the new digest generated?

Reply:
It's generated using some procedural macros that build a type graph and hash the output. I just ran `./test-abi.sh` locally to see what the mismatch was and updated the value.